pkg/apis/core/v1/conversion_test.go

```diff
@@ -419,3 +419,65 @@ func Test_v1_PodStatus_to_core_PodStatus(t *testing.T) {
 		t.Errorf("Convert v1.PodStatus to core.PodStatus failed. expected corePodStatus.PodIPs[0].ip=%v found %v", v1PodStatus.PodIP, corePodStatus.PodIPs[0].IP)
 	}
 }
+
+func Test_core_NodeSpec_to_v1_NodeSpec(t *testing.T) {
+	// core to v1
+	var v1NodeSpec v1.NodeSpec
+	coreNodeSpec := core.NodeSpec{
+		PodCIDRs: []string{"10.0.0.0/24", "10.0.1.0/24"},
+	}
+	// convert
+	if err := corev1.Convert_core_NodeSpec_To_v1_NodeSpec(&coreNodeSpec, &v1NodeSpec, nil); nil != err {
```

Comment:
We should have a table of test cases including no entries, 1 entries, and > 1 entry

Reply:
083bf4e 

adds more tests. Please check..